### PR TITLE
[Enhancement] Updates title of psh environment when pr is from fork

### LINKED ISSUE #3101: [Enhancement] Add PR title to environment name for PRs from forks
### Where on docs.platform.sh should be changed?

https://github.com/platformsh/platformsh-docs/blob/c6c980f4d25e381261c88cc47826bb5227f21350/.github/workflows/build-from-fork.yaml#L65

### What exactly should be updated?

Add the pull request title to PR environments that come from forks. 

PR from docs repo:
```
pr-3055             | (PR-3055) Add section about log forwarding level mapping
```
vs PR from fork:
```
bffpr-3100          | bffpr-3100  
```

### Additional context

We should be able to add the title right before we activate the environment. 

Initially, the environment title will **not** be in sync with changes to the PR title. That come in phase 2.

### PR DESCRIPTION
**Pull request by gilzow**
## Why

Closes #3101 

## What's changed

- updates the shortcut `env` to `environment` in the platform command
- adds a section for updating the title of the platformsh environment before we activate it. 
- Closes #3101

### CODE REVIEW

File: .github/workflows/build-from-fork.yaml [hunk 1]
```diff
@@ -60,7 +60,11 @@ jobs:
...
 
           # If environment not active, activate it
-          if ! $(platform env --format plain --columns title,status | grep '${{ env.BRANCH_TITLE }}' | grep -q Active); then
-            echo "Activating environment"
+          if ! $(platform environment --format plain --columns title,status | grep '${{ env.BRANCH_TITLE }}' | grep -q Active); then
+            echo "::notice::Updating Platform.sh environment title for better tracking..."
+            # in this case we DO want to staticaly include `pr-##` since we're referring to a pull request number
+            newTitle="(pr-${{ github.event.number }}) ${{ github.event.pull_request.title }}"
+            platform environment:info title "${newTitle}" -e ${{ env.BRANCH_TITLE }}
+            echo "::notice::Activating environment ${{ env.BRANCH_TITLE }}"
```
Comment (chadwcarlson) [inline .github/workflows/build-from-fork.yaml:68]: @gilzow Do we want to do this same thing here so everything is the same?

https://github.com/platformsh/platformsh-docs/blob/5c46a76de852959699df096977543e164c01774d/.github/workflows/manage-environment.yaml#L16-L35

Comment (chadwcarlson) [reply .github/workflows/build-from-fork.yaml:68]: cc @AnouckColson - thoughts on your side?

Comment (AnouckColson) [reply .github/workflows/build-from-fork.yaml:68]: @gilzow @chadwcarlson 👍 I'm all in favour of consistency.

Comment (gilzow) [reply .github/workflows/build-from-fork.yaml:68]: Added. For now I added it in the other workflow, but at some point I want to re-evaluate these workflows to see if we can make them more streamlined.

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**